pages/agent/v3/ssh_keys.md.erb

```diff
@@ -123,3 +123,19 @@ set -eu
 
 export SSH_AUTH_SOCK="/var/lib/buildkite-agent/.ssh/ssh-agent.sock"
 ```
+
+## Using multiple keys by adding pipeline-specific environments  
+
+There is yet another way to use multiple SSH keys.  
+
+1. Add a pipeline-specific environment (for example, by using [Elastic Stack's secrets support](https://github.com/buildkite/elastic-ci-stack-for-aws#build-secrets) or by having an Agent environment hook that switches on the repository URL or the pipeline slug):  
+```bash
+GIT_SSH_COMMAND="ssh -i ~/.ssh/id_rsa_mypipeline"
+```
+2. Create an identity file at that location:
+```bash
+~/.ssh/id_rsa_mypipeline
+```
+3. Add the public key for that identity file to `mypipeline` on the git repository provider
```

Comment:
I'm not :100: sure what this means.

Through out we're using "mypipeline" as something the reader needs to replace? Let's say that to beging with. Then what are we adding the identity to? The account associated to the repo associated to the pipeline? And calling it "mypipeline" ?